doc/input-output-reference/src/overview/api-usage.tex

```diff
@@ -10,23 +10,76 @@ \section{Using EnergyPlus as a Library}\label{sec:api-usage}
 
 \begin{itemize}
  \item The Energy Management System, which was the first method to allow reading and writing simulation data while a simulation was running, and allowing user-defined scripts to be executed to alter simulation data.
- \item The EnergyPlus unit test structure, which has, as a side-effect, enabled the ability to "reset" the state of a simulation, and re-run another, in the same memory space.
+ \item The EnergyPlus unit test structure, which has, as a side-effect, enabled the ability to ``reset'' the state of a simulation, and re-run another, in the same memory space.
```

Comment:
Full overhaul of the API documentation here, updated with the state changes along with other improved information.